x-pack/plugins/infra/public/plugin.ts

```diff
@@ -308,9 +310,11 @@ export class Plugin implements InfraClientPluginClass {
         const [coreStart, plugins, pluginStart] = await core.getStartServices();
         const { renderApp } = await import('./apps/metrics_app');
 
+        const isCloudEnabled = pluginsSetup.cloud?.isCloudEnabled || !!pluginsSetup.cloud?.cloudId;
+        const isServerlessEnabled = pluginsSetup.cloud?.isServerlessEnabled || this.isServerlessEnv;
         return renderApp(
           coreStart,
-          { ...plugins, kibanaVersion: this.kibanaVersion },
+          { ...plugins, kibanaVersion: this.kibanaVersion, isCloudEnabled, isServerlessEnabled },
```

Comment:
The naming for `isCloudEnabled` and `isServerlessEnabled` seems a bit misleading in the `cloud` plugin. They look like feature flags within the app but in reality describe the surrounding environment. The one that you named `isServerlessEnv` feels more to the point, might be worth sticking to similar names when passing them to Infra. Wdyt?

Reply:
I wanted to keep it consistent with the naming in the cloud plugin so it's easier to find where those flags are coming from. What would you suggest: `isServerlessEnv` and i`sCloudEnv`? I am open to ideas in case we don't want to have the same naming as the one `cloud` plugin provides

Reply:
Yeah, I think `isServerlessEnv` and `isCloudEnv` would read better in the code. Maybe I don't get the intention that folks in the `cloud` plugin had with those flag names but for our purpose `*Env` suffix seems to work better

Reply:
Renamed ✅